CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ option(USE_PKCS11_HELPER_LIBRARY "Build mbed TLS with the pkcs11-helper library.
 option(ENABLE_ZLIB_SUPPORT "Build mbed TLS with zlib library." OFF)
 
 option(ENABLE_PROGRAMS "Build mbed TLS programs." ON)
+option(USE_CRYPTO_SUBMODULE "Build and use libmbedcrypto from the crypto submodule." OFF)
```

Comment:
[Not a blocker] I think we need to state that `USE_CRYPTO_SUBMODULE ` is a temporary option, and we will be moving the Crypto to the submodule and this will be not just the default, but the only option.